internal/meta/repository.go

```diff
@@ -0,0 +1,50 @@
+package meta
+
+import (
+	"emperror.dev/errors"
+	"encoding/json"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+type Repository struct {
+	// The GitHub (GraphQL) ID of the repository (e.g., R_kgDOHMmHmg).
+	ID string `json:"id"`
+	// The owner of the repository (e.g., aviator-co)
+	Owner string `json:"owner"`
+	// The name of the repository (e.g., av)
+	Name string `json:"name"`
+}
+
+func GetRepository(repo *git.Repo) (Repository, bool) {
```

Comment:
prefer if we switch to Read* and Write* for meta, vs Get* and Create* for API calls and other actions.